smalltalksrc/VMMaker/StackInterpreter.class.st

```diff
@@ -14736,6 +14736,7 @@ StackInterpreter >> stackPointerIndexForFrame: theFP [
 	<var: #theFP type: #'char *'>
 	<var: #thePage type: #'StackPage *'>
 	<var: #theSP type: #'char *'>
+	<returnTypeC: #sqInt>
```

Comment:
Oh yes :)

Is this a problem with the type inferencer? Something worth improving there as a separate issue?

Reply:
It's not just the type inferer, the problem is to have char* types in pointers to the stack (the type inference is doing good it's work).

I already opened an issue for that.